### DISCUSSION TITLE
How to remove Prefixes/Suffixes from Essential Messages

### DISCUSSION THREAD

**Discussion by JHiroshige** [EssentialsX/Essentials#4070] [Q&A] [answered]
So Im trying to remove the Prefixes/Suffixes from Essentials Messages. 
Example: 
![image](https://user-images.githubusercontent.com/67478052/111844598-9f923b00-8903-11eb-8844-ed992c26df28.png)

The ADMIN and hex code is the prefix
I already asked in the MOSS Discord. No responses or none of them helped/worked

I already removed the suffixes in the config.yml
Im using ChatControlPro for my Chat

Server: 1.16.4 Paper

**Comment by triagonal** [comment, marked as answer]
EssentialsX uses the player's displayname for all "system" messages. There's currently no way to toggle between using regular usernames and displaynames. Follow #1406 for updates on this feature.

EssentialsX only modifies the displayname if you have `change-displayname` set to `true` in your config, so make sure that's set to `false`. For good measure, also set `add-prefix-suffix` to `false` in your config.

If that doesn't work, then your chat plugin or another plugin on your server is modifying the player's displayname and you'll need to find out how to disable it.
